### ISSUE TITLE
Django 2.0

### ISSUE THREAD

**Issue by noperson83** [bitlabstudio/django-calendarium#61]
Does calendarium work with Django 2.0??

**Comment by Tyrdall** [comment]
Give it a try!

**Comment by jmn** [comment]
I tried it, got this error:
```
calendarium\models.py", line 15, in <module> 
ModuleNotFoundError: No module named 'django.core.urlresolvers
```

**Comment by Tyrdall** [comment]
@jmn Yup, it's not ready for 2.0 yet: https://github.com/bitlabstudio/django-calendarium#compatibility

**Comment by noperson83** [comment]
I got it to work with 2.0. 
I am having issues with the week view displaying the occurrence one day behind?
on the month view, they display on the correct day..

**Comment by Tony363** [comment]
i also got it to work with python 2.0. but now, im having problems with "ModuleNotFoundError: No module named 'django.utils.six'" after i tried to update relevant imports again

**Comment by stonks1334** [comment]
Is this project still being maintained or is it abandon-ware?

**Comment by mbrochh** [comment]
We don't actively work on our packages any more. If you submit a patch, we will merge it. If you would like to take over maintenance, let us know.
